projects/demo-integrations/cypress/tests/core/dialogs/dialogs.cy.ts

```diff
@@ -75,32 +75,51 @@ describe(`Dialogs`, () => {
             });
 
             it(`Dialog with confirmation works`, () => {
-                cy.get(`tui-dialog-example-8 button`).first().click();
-
-                cy.tuiHide(`tui-doc-page`);
-                cy.tuiWaitKitDialog();
-
-                cy.get(`tui-dialog`)
-                    .last()
-                    .tuiWaitBeforeScreenshot()
-                    .matchImageSnapshot(`${index}-7-dialogs`);
-
-                cy.get(`tui-dialog .t-close`).click();
-                cy.tuiWaitKitDialog();
-
-                cy.get(`tui-dialog`)
-                    .last()
-                    .tuiWaitBeforeScreenshot()
-                    .matchImageSnapshot(`${index}-7-dialogs-prompt`);
-
-                cy.get(`tui-prompt .t-button`).last().click();
-
-                cy.tuiShow(`tui-doc-page`);
-
-                cy.tuiWaitBeforeScreenshot().matchImageSnapshot(
-                    `${index}-7-dialogs-prompt-gone`,
-                    {capture: `viewport`},
-                );
+                beforeEach(() => {
```

Comment:
<img width="1410" alt="image" src="https://user-images.githubusercontent.com/12021443/224723674-a6379e7f-4995-4097-a2a0-6ed627df8979.png">

rename 
```
it(`Dialog with confirmation works`, () => {
``` to 

```
describe(`Dialog with confirmation works`, () => {
```